Sources/RedCat/CombineStore.swift

```diff
@@ -0,0 +1,118 @@
+//
+//  CombineStore.swift
+//  
+//
+//  Created by Markus Pfeifer on 15.05.21.
+//
+
+import Foundation
+
```

Comment:
There should be special treatment in case that there's only one observer, because property lookup is faster than dictionary lookup.